src/box/tuple_compare.cc

```diff
@@ -572,6 +572,60 @@ tuple_compare_field_with_type(const char *field_a, enum mp_type a_type,
 	}
 }
 
+/*
+ * Implements the field comparison logic. If the key we use is not nullable
+ * then a simple call to tuple_compare_field is used.
+ *
+ * Othervice one of \p field_a and \p field_b can be NIL: either it's encoded
```

Comment:
Typo: otherwice -> otherwise. Also, there's a typo in the commit log: siplify -> simplify

Reply:
Fixed.